lra/coordinator/client/narayana-client/src/main/java/io/helidon/lra/coordinator/client/narayana/LRAStatusReader.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.lra.coordinator.client.narayana;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.concurrent.Flow;
+import java.util.stream.Collectors;
+
+import io.helidon.common.GenericType;
+import io.helidon.common.http.DataChunk;
+import io.helidon.common.reactive.Multi;
+import io.helidon.common.reactive.Single;
+import io.helidon.media.common.MessageBodyReader;
+import io.helidon.media.common.MessageBodyReaderContext;
+
+import org.eclipse.microprofile.lra.annotation.LRAStatus;
+
+class LRAStatusReader implements MessageBodyReader<LRAStatus> {
```

Comment:
When using abbreviations that are all upper case, we lowercase them in class names in Helidon.
So the class name should be `LraStatusReader` (valid for any usage in this PR)

Reply:
Fixed